### ISSUE TITLE
Evaluation mode in Resnet

### ISSUE THREAD

**Issue by zhangmozhe** [yunjey/pytorch-tutorial#62]
I have a question about the evaluation mode. I found that in the resnet tutorial, the network is not set to evaluation through resnet.eval(). Will this affect the testing accuracy? Thanks!

**Comment by yunjey** [comment]
@zhangmozhe Yes, it hurts the testing accuracy. If you use `resnet.eval()`, batch normalization layer uses running average/variance instead of mini-batch statistics. You can improve the performance when using `resnet.eval()` by changing the momentum coefficient in batch normalization layer.

It is recommended to change `nn.BatchNorm2d(16) ` to `nn.BatchNorm2d(16, momentum=0.01)`. The default value of the momentum is 0.1.
